_data/people/dan131riley.yml

```diff
@@ -64,3 +64,11 @@ presentations:
   location: Virtual
   focus-area: ia
   project: mkfit
+- title: Accelerated Strip Local reconstruction on GPUs
+  date: 2021-06-24
+  url: https://indico.cern.ch/event/992968/contributions/4421875/attachments/2270766/3856423/gpu-unpacking-2021-06-24.pdf
```

Comment:
Could you link to the actual indico entry, not the notebook? The indico link is more useful and also has this link, and the URL is stable. Thanks.

Reply:
@dan131riley Ping?